ramls/order.raml

```diff
@@ -51,7 +52,33 @@ resourceTypes:
         example:
           strict: false
           value: !include acq-models/examples/composite_purchase_order.sample
-
+  /po_number/validate:
+     type:
+        post-item:
+          exampleItem: !include acq-models/mod-orders-storage/examples/po_number_get.sample
+          schema: po_number
+      
+     displayName: PO Number Validation
+     post:
+        description: validate if the PO Number is unique and matches the pattern specified
+        is: [validate, auth]
+        body:
+          application/json:
+            type: po_number
+        responses:
+          204:
+            description: "Valid PO_Number"
+          400:
+            description: "Bad request, e.g. malformed request body or query parameter. Details of the error (e.g. name of the parameter or line/character number with malformed data) provided in the response."
```

Comment:
Maybe update this description as it seems to be a little misleading to me.  I think a malformed request would actually result in a 422 most of the time.  A 400 would occur when the po_number provided already exists.

Reply:
Added valid description